### DISCUSSION TITLE
First milestone towards 1.0.0

### DISCUSSION THREAD

**Discussion by mcculls** [eclipse-sisu/sisu-project#56] [Announcements]
## What's Changed
* Fix CDI related issues by @cstamas in https://github.com/eclipse/sisu.inject/pull/1
* Build with Eclipse/Tycho 2.5.0 and Java 11 by @mcculls in https://github.com/eclipse/sisu.inject/pull/33
* Raise problem reporting logs to DEBUG, fixes #36 by @gnodet in https://github.com/eclipse/sisu.inject/pull/37
* Upgrade internal copy of ASM to 9.2 by @mcculls in https://github.com/eclipse/sisu.inject/pull/34
* Implement PathTypeConverter by @mcculls in https://github.com/eclipse/sisu.inject/pull/39
* Add JUnit 5 annotations to InjectedTest setUp/tearDown by @mcculls in https://github.com/eclipse/sisu.inject/pull/40
* Fix static parameters binding lookup by @mcculls in https://github.com/eclipse/sisu.inject/pull/41
* Run injection tests against multiple versions of Guice by @mcculls in https://github.com/eclipse/sisu.inject/pull/42
* Support using @Priority on Providers by @mcculls in https://github.com/eclipse/sisu.inject/pull/43
* Use read lock when subscribing to publishers… by @mcculls in https://github.com/eclipse/sisu.inject/pull/44
* Cache binding lookups for single bean providers by @mcculls in https://github.com/eclipse/sisu.inject/pull/45
* Use AtomicReferenceFieldUpdater as it works better for large numbers of instances by @mcculls in https://github.com/eclipse/sisu.inject/pull/46
* Enable Java CI workflow by @mcculls in https://github.com/eclipse/sisu.inject/pull/47
* Enable CodeQL analysis by @mcculls in https://github.com/eclipse/sisu.inject/pull/48
* Replace potentially-expensive regex with simple tokenizer by @mcculls in https://github.com/eclipse/sisu.inject/pull/49
* Allow Main to boot with extra bindings by @bentmann in https://github.com/eclipse/sisu.inject/pull/50
* Re-enable various resource-related unit tests by @mcculls in https://github.com/eclipse/sisu.inject/pull/51
* Rework globber pattern strategy to avoid use of regex by @mcculls in https://github.com/eclipse/sisu.inject/pull/52
* Use GlobberStrategy.PATTERN instead of regex for ServiceBindings filtering by @mcculls in https://github.com/eclipse/sisu.inject/pull/53

## New Contributors
* @cstamas made their first contribution in https://github.com/eclipse/sisu.inject/pull/1
* @gnodet made their first contribution in https://github.com/eclipse/sisu.inject/pull/37
* @bentmann made their first contribution in https://github.com/eclipse/sisu.inject/pull/50

**Full Changelog**: https://github.com/eclipse/sisu.inject/commits/milestones/0.9.0.M1

<hr /><em>This discussion was created from the release <a href='https://github.com/eclipse/sisu.inject/releases/tag/milestones/0.9.0.M1'>First milestone towards 1.0.0</a>.</em>

**Comment by j-s-3** [comment]
Hey @mcculls there are changes in 0.9.0.M2 that we'd like to make use of but we're not sure whether it's safe to be using a milestone release. I'm leaning towards it being fine given this project has been relatively stable but thought I better check with you first.

**Comment by mcculls** [comment]
Hi @jlstephens89 yes please go ahead and use 0.9.0.M2 - although it is a milestone, it has gone through a lot of testing
